src/app/app.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from './services/user.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent implements OnInit{
+  public senderName: string = this.userService.userData[0].name;
+  public receiverName: string = this.userService.userData[1].name;
+  public senderColour: any = localStorage.getItem(this.senderName);
```

Comment:
what if the item does not exist ?

Reply:
I have initialized this in ngOnInit(), Chandu